xendit/models/ewallet/ewallet.py

```diff
@@ -237,3 +282,90 @@ def _parse_ewallet_type(ewallet_type):
             return ewallet_type.name
         except AttributeError:
             return ewallet_type
+
+    @staticmethod
+    def create_ewallet_charge(
+        *,
+        reference_id,
+        currency,
+        amount,
+        checkout_method,
+        channel_code=None,
+        channel_properties=None,
+        customer_id=None,
+        basket=None,
+        metadata=None,
+        for_user_id=None,
+        with_fee_rule=None,
+        **kwargs,
+    ):
+        """Send POST Request to create EWallet Charge (API Reference: eWallets/Create E-Wallet Charge)
+
+        Args:
+          - reference_id (str)
+          - currency (str)
+          - amount (int)
+          - checkout_method (str)
+          - channel_code (str)
+          - channel_properties (dict)
+          - customer_id (str)
+          - basket (list)
+          - metadata (dict)
+          - **for_user_id (str) (XenPlatforms only)
+          - **with_fee_rule (str) (XenPlatforms only)
```

Comment:
same argument like above
```suggestion
        *,
        reference_id: str,
        currency: str,
        amount: int,
        checkout_method: str,
        channel_code: str=None,
        channel_properties:dict =None,
        customer_id: str=None,
        basket:list =None,
        metadata: dict =None,
        for_user_id: str =None,
        with_fee_rule: str =None,
        **kwargs,
    ):
        """Send POST Request to create EWallet Charge (API Reference: eWallets/Create E-Wallet Charge)

        Args:
          - reference_id (str)
          - currency (str)
          - amount (int)
          - checkout_method (str)
          - channel_code (str)
          - channel_properties (dict)
          - customer_id (str)
          - basket (list)
          - metadata (dict)
          - **for_user_id (str) (XenPlatforms only)
          - **with_fee_rule (str) (XenPlatforms only)
```